src/renderer/components/wallet/txs/send/SendFormBCH.tsx

```diff
@@ -0,0 +1,390 @@
+          () =>
+            // render radio group while reloading fees
+            FP.pipe(
+              prevFeesWithRatesRef.current,
+              O.map(renderFeeOptionsRadioGroup),
+              O.getOrElse(() => <></>)
+            ),
+          renderFeeOptionsRadioGroup
+        )
+      ),
+    [prevFeesWithRatesRef, oFeesWithRates, renderFeeOptionsRadioGroup]
+  )
+
+  const addressValidator = useCallback(
+    async (_: unknown, value: string) => {
+      if (!value) {
+        return Promise.reject(intl.formatMessage({ id: 'wallet.errors.address.empty' }))
+      }
+      if (!addressValidation(value.toLowerCase())) {
+        return Promise.reject(intl.formatMessage({ id: 'wallet.errors.address.invalid' }))
+      }
+    },
+    [addressValidation, intl]
+  )
+
+  const maxAmount: BaseAmount = useMemo(
+    () =>
+      FP.pipe(
+        selectedFee,
+        O.alt(() => prevSelectedFeeRef.current),
+        O.map((fee) => balance.amount.amount().minus(fee.amount())),
+        // Set maxAmount to zero as long as we dont have a feeRate
+        O.getOrElse(() => ZERO_BN),
+        baseAmount
+      ),
+    [balance.amount, selectedFee]
+  )
+
+  useEffect(() => {
+    // Whenever `amountToSend` has been updated, we put it back into input field
+    form.setFieldsValue({
+      amount: baseToAsset(amountToSend).amount()
+    })
+  }, [amountToSend, form])
+
+  const amountValidator = useCallback(
+    async (_: unknown, value: BigNumber) => {
+      // error messages
+      const errors = {
+        msg1: intl.formatMessage({ id: 'wallet.errors.amount.shouldBeNumber' }),
+        msg2: intl.formatMessage({ id: 'wallet.errors.amount.shouldBeGreaterThan' }, { amount: '0' }),
+        msg3: intl.formatMessage({ id: 'wallet.errors.amount.shouldBeLessThanBalanceAndFee' })
+      }
+      return validateTxAmountInput({ input: value, maxAmount: baseToAsset(maxAmount), errors })
+    },
+    [intl, maxAmount]
+  )
+
+  // State for visibility of Modal to confirm tx
+  const [showPwModal, setShowPwModal] = useState(false)
+
+  const sendHandler = useCallback(() => {
+    // close PW modal
+    setShowPwModal(false)
+
+    onSubmit({
+      recipient: form.getFieldValue('recipient'),
+      asset: balance.asset,
+      amount: assetToBase(assetAmount(form.getFieldValue('amount'))),
+      feeOptionKey: selectedFeeOptionKey,
+      memo: form.getFieldValue('memo'),
+      txType: TxTypes.TRANSFER
+    })
+  }, [selectedFeeOptionKey, onSubmit, form, balance])
+
+  const renderPwModal = useMemo(
+    () =>
+      showPwModal ? (
+        <PasswordModal
+          onSuccess={sendHandler}
+          onClose={() => setShowPwModal(false)}
+          validatePassword$={validatePassword$}
+        />
+      ) : (
+        <></>
+      ),
+    [sendHandler, showPwModal, validatePassword$]
+  )
+
+  const uiFeesRD: UIFeesRD = useMemo(
+    () =>
+      FP.pipe(
+        feeRD,
+        RD.map((fee) => [{ asset: AssetBTC, amount: fee }])
```

Comment:
same question about BTC asset

Reply:
Yep, BCH should be there, thx for pointing it out!